src/BaseHtml.php

```diff
@@ -1,68 +1,57 @@
 <?php
-/**
- * @link http://www.yiiframework.com/
- * @copyright Copyright (c) 2008 Yii Software LLC
- * @license http://www.yiiframework.com/license/
- */
+declare(strict_types = 1);
 
 namespace Yiisoft\Yii\Bootstrap4;
 
 use Yiisoft\Arrays\ArrayHelper;
 
 /**
- * BaseHtml provides concrete implementation for [[Html]].
+ * BaseHtml provides concrete implementation for {@see Html}.
  *
- * Do not use BaseHtml. Use [[Html]] instead.
- *
- * @author Paul Klimov <klimov.paul@gmail.com>
+ * Do not use BaseHtml. Use {@see Html} instead.
  */
-class BaseHtml extends \yii\helpers\Html
+class BaseHtml extends \Yiisoft\Html\Html
 {
-
-    /**
-     * @var int a counter used to generate [[id]] for widgets.
-     * @internal
-     */
-    public static $counter = 0;
-    /**
-     * @var string the prefix to the automatically generated widget IDs.
-     * @see getId()
-     */
-    public static $autoIdPrefix = 'i';
     /**
      * @var array list of tag attributes that should be specially handled when their values are of array type.
-     * In particular, if the value of the `data` attribute is `['name' => 'xyz', 'age' => 13]`, two attributes
-     * will be generated instead of one: `data-name="xyz" data-age="13"`.
-     * @since 2.0.3
+     *
+     * In particular, if the value of the `data` attribute is `['name' => 'xyz', 'age' => 13]`, two attributes will be
+     * generated instead of one: `data-name="xyz" data-age="13"`.
      */
     public static $dataAttributes = ['data', 'data-ng', 'ng', 'aria'];
 
     /**
      * Renders Bootstrap static form control.
      *
      * @param string $value static control value.
-     * @param array $options the tag options in terms of name-value pairs. These will be rendered as
-     * the attributes of the resulting tag. There are also a special options:
+     * @param array $options the tag options in terms of name-value pairs. These will be rendered as the attributes of
+     *                       the resulting tag. There are also a special options:
```

Comment:
```suggestion
     * the resulting tag. There are also a special options:
```

Reply:
Do we have indents in all the packages in which the variables are defined, are they going to be removed?